src/java.prefs/macosx/classes/java/util/prefs/MacOSXPreferencesFile.java

```diff
@@ -82,15 +82,9 @@ class MacOSXPreferencesFile {
         loadPrefsLib();
     }
 
-    @SuppressWarnings({"removal", "restricted"})
+    @SuppressWarnings("restricted")
     private static void loadPrefsLib() {
-        java.security.AccessController.doPrivileged(
-            new java.security.PrivilegedAction<Void>() {
-                public Void run() {
-                    System.loadLibrary("prefs");
-                    return null;
-                }
-            });
+        System.loadLibrary("prefs");
```

Comment:
What am I missing?  Why keep the private static method if it only contains a single method invocation and is only used in the static initializer?

Reply:
We want to `@SuppressWarnings("restricted")` the `loadLibrary()` call. IIUC, I can't `@SuppressWarnings` the static block, nor the call to `loadLibrary()`, so AFAICT that call needs to be in a separate, `@SuppressWarnings`-ed method.

Reply:
Right. The other option is to put the annotation above the class definition which would cover the entire class, but I think this is better as it makes the annotation specific to that method only.